src/views/HomeView.vue

```diff
@@ -61,6 +61,7 @@ async function submitForm(validationErrors: FormValidationError) {
   } finally {
     if (authenticatedUserResponse.value) {
       userStore.setUserStorageCrendentials(authenticatedUserResponse.value)
+      userStore.fetchUser()
```

Comment:
We now call the `fetchUser` Pinia action right after we set local storage and update the store.